src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -263,12 +273,20 @@ ucs_status_t uct_rc_verbs_ep_get_zcopy(uct_ep_h tl_ep, const uct_iov_t *iov, siz
                                        uint64_t remote_addr, uct_rkey_t rkey,
                                        uct_completion_t *comp)
 {
-    uct_rc_verbs_ep_t *ep = ucs_derived_of(tl_ep, uct_rc_verbs_ep_t);
+    uct_base_iface_t *iface = NULL;
+    uct_rc_verbs_ep_t *ep = NULL;
     ucs_status_t status;
+    size_t length = 0;
+
+    iface = ucs_derived_of(tl_ep->iface, uct_base_iface_t);
+    ep    = ucs_derived_of(tl_ep, uct_rc_verbs_ep_t);
 
-    UCT_CHECK_PARAM_IOV(iov, iovcnt, buffer, length, memh);
+    UCT_CHECK_PARAM(iovcnt <= iface->config.max_iov,
```

Comment:
@yosefe UCT_CHECK_PARAM_IOV is temporally macro that provide required local variables `buffer`,  `length`, `memh`. These variables aren't used in iov implementation.